_data/patterns/race.yml

```diff
@@ -0,0 +1,88 @@
+legend: |
+  What is your race and/or ethnicity?
+hint: |
+  Select all that apply
```

Comment:
I put the content in the race.yml file because I wanted to consolidate all of the content into a single location. (I know this isn't entirely pure in this PR, but I liked that the text was next to the other hint text) I've made a couple of improvements to improve clarity, but if you want me to add the hint text directly to the include, let me know!